From a2ba50b6a47600f7108e0166f2b3b4a8b351f62c Mon Sep 17 00:00:00 2001
From: Indy Lin <indy_lin@tw.shuttle.com>
Date: Fri, 17 Jun 2016 14:16:32 +0800
Subject: [PATCH 139/443] In Dashboard, fix offline devices count calculation
 error.

---
 dashboard/views.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dashboard/views.py b/dashboard/views.py
index a51eb51..c50b08b 100644
--- a/dashboard/views.py
+++ b/dashboard/views.py
@@ -77,7 +77,8 @@ def dashboard_views(request):
 
     device_total_count = Devices.objects.count()
     device_proposed_count = Devices.objects.filter(State='Proposed').count()
-    device_offline_count = Devices.objects.filter(State='Offline').count()
+    offline_time_check = now - timedelta(seconds = 30)
+    device_offline_count = Devices.objects.filter(UpdateDate__lt=offline_time_check).count()
     device_online_count = device_total_count - device_offline_count - device_proposed_count
 
     devices_query_set = Devices.objects.all().order_by('-UpdateDate')
-- 
1.9.1

