From 0979057e114cebfc91896c7ab304b7e0c22440a3 Mon Sep 17 00:00:00 2001
From: Indy Lin <indy_lin@tw.shuttle.com>
Date: Wed, 6 Jul 2016 17:38:53 +0800
Subject: [PATCH 167/443] Avoid duplication # comment sign to write to crontab
 file.

---
 systemsettings/views.py | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/systemsettings/views.py b/systemsettings/views.py
index f29d2da..a2607e7 100644
--- a/systemsettings/views.py
+++ b/systemsettings/views.py
@@ -534,10 +534,12 @@ def settings_system_backup(request):
 
         if BackupMethod == 'Manually':
             if content2_idx != -1:
-                #print('old content2:'+ content2[0])
-                content2[0] = '#'+ content2[0]
-                #print('new content2:'+ content2[0])
-                content[content2_idx]=content2[0]
+                cmd = content2[0]
+                #print('old cmd:'+ cmd)
+                if cmd[0] != '#':
+                    cmd = '#'+ cmd
+                    #print('new cmd:'+ cmd)
+                    content[content2_idx]=cmd
 
             #os.system('{0}\n'.format(backupshfile))
             os.system('cd {0} && {1}\n'.format(settings.BASE_DIR, backupshfile))
-- 
1.9.1

