From 541808d465dbf4bf047eec077b6bc5193c3a1716 Mon Sep 17 00:00:00 2001
From: Wellens <wellens@tw.shuttle.com>
Date: Fri, 5 Aug 2016 11:57:55 +0800
Subject: [PATCH 227/443] Update the version to V.0.1.a.20160805

---
 br06/settings.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/br06/settings.py b/br06/settings.py
index 3a7a096..a6d47c2 100644
--- a/br06/settings.py
+++ b/br06/settings.py
@@ -13,7 +13,7 @@ https://docs.djangoproject.com/en/dev/ref/settings/
 import os
 from api.utility import get_localtimezone
 
-VERSION = 'V.0.1.a.20160804'
+VERSION = 'V.0.1.a.20160805'
 print(VERSION)
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-- 
1.9.1

