From c2b00cbf86e0d6eb1a37c0c20a8493ee2286db08 Mon Sep 17 00:00:00 2001
From: Jeffery <jeffery_wu@tw.shuttle.com>
Date: Thu, 1 Sep 2016 14:29:57 +0800
Subject: [PATCH 249/443] Update the version to V.0.1.a.20160903

---
 br06/settings.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/br06/settings.py b/br06/settings.py
index b38e774..8184dcb 100644
--- a/br06/settings.py
+++ b/br06/settings.py
@@ -13,7 +13,7 @@ https://docs.djangoproject.com/en/dev/ref/settings/
 import os
 from api.utility import get_localtimezone
 
-VERSION = 'V.0.1.a.20160826'
+VERSION = 'V.0.1.a.20160902'
 print(VERSION)
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
@@ -190,7 +190,7 @@ DEFAULT_FROM_EMAIL = 'fr_serivce@tw.shuttle.com'
 SESSION_EXPIRE_AT_BROWSER_CLOSE = True
 SESSION_COOKIE_AGE = 86400 # sec
 
-# 4G   
+# 4G
 EN_4G = 0 # Enable 4Gapi in UserAdd
 SE_4G = 1 # 4G return status: success
 LO_4G = 'http://61.247.173.118/daily/LoginApi/face'
-- 
1.9.1

