From f0be898b90ab6ef4586fdbc5fb28faa27837881c Mon Sep 17 00:00:00 2001
From: Jeffery <jeffery_wu@tw.shuttle.com>
Date: Thu, 1 Sep 2016 16:13:23 +0800
Subject: [PATCH 252/443] Re-package BR06 SW V.0.1.a.20160902 code

---
 br06/settings.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/br06/settings.py b/br06/settings.py
index 35337a9..8184dcb 100644
--- a/br06/settings.py
+++ b/br06/settings.py
@@ -16,7 +16,6 @@ from api.utility import get_localtimezone
 VERSION = 'V.0.1.a.20160902'
 print(VERSION)
 
-
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 PROJECT_DIR = os.path.dirname(__file__)
-- 
1.9.1

