From 164abe70f184a9d1514740b2eccd02cc3a87309d Mon Sep 17 00:00:00 2001
From: Jeffery <jeffery_wu@tw.shuttle.com>
Date: Fri, 30 Sep 2016 19:15:00 +0800
Subject: [PATCH 303/443] Update the version to V.0.2.a.20160930

---
 br06/settings.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/br06/settings.py b/br06/settings.py
index f95e30b..707e473 100644
--- a/br06/settings.py
+++ b/br06/settings.py
@@ -13,7 +13,7 @@ https://docs.djangoproject.com/en/dev/ref/settings/
 import os
 from api.utility import get_localtimezone
 
-VERSION = 'V.0.1.a.20160914'
+VERSION = 'V.0.2.a.20160930'
 print(VERSION)
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-- 
1.9.1

