From 254f8906cc66b2c81c7b2c26427db2f59db94a37 Mon Sep 17 00:00:00 2001
From: Wellens <wellens@tw.shuttle.com>
Date: Tue, 1 Nov 2016 17:21:57 +0800
Subject: [PATCH 342/443] Fix bug - log alway prints create frusergroup.

---
 frusergroups/views.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/frusergroups/views.py b/frusergroups/views.py
index 6b4ff96..26cacdb 100755
--- a/frusergroups/views.py
+++ b/frusergroups/views.py
@@ -12,8 +12,8 @@ res_code = [{"CODE": "0"}, {"CODE": "1"}, {"CODE": "2"}, {"CODE": "3"}]
 permission = 'permission_frusergroup'
 
 try:
-    group, group_exists = Frusergroups.objects.get_or_create(Name = 'Default')
-    if not group_exists:
+    group, group_created = Frusergroups.objects.get_or_create(Name = 'Default')
+    if group_created:
         print('Create default FR user group.')
     else:
         if settings.DEBUG:
-- 
1.9.1

