From 49d8836930d69f1aa9a1d175907960dafc357263 Mon Sep 17 00:00:00 2001
From: Wellens <wellens@tw.shuttle.com>
Date: Tue, 13 Dec 2016 16:21:32 +0800
Subject: [PATCH 411/443] Modify default FTP Account and password.

---
 config.ini | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.ini b/config.ini
index 3a6f195..4fb147e 100755
--- a/config.ini
+++ b/config.ini
@@ -23,8 +23,8 @@ FTP_PATH =
 FTP_FILE_NAME = update.json
 FTP_HOST = ftp.tw.shuttle.com
 FTP_PORT = 21
-FTP_USER = client.br06@swdesign
-FTP_PW = download
+FTP_USER = Client.BR06.STD@swdesign
+FTP_PW = BR06.STD
 FTP_DEBUG_LEVEL = 0
 
 [Face_score]
-- 
1.9.1

