From 4fe9d3e8d9545150fd14492bd3f63e256aadd012 Mon Sep 17 00:00:00 2001
From: Wellens <wellens@tw.shuttle.com>
Date: Wed, 21 Dec 2016 14:14:15 +0800
Subject: [PATCH 434/443] Update the version to V.1.0.a.20161221

Entry MP.
---
 br06/settings.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/br06/settings.py b/br06/settings.py
index 89dcf72..828f353 100755
--- a/br06/settings.py
+++ b/br06/settings.py
@@ -2,7 +2,7 @@ import os
 from api.utility import get_localtimezone
 from configparser import ConfigParser
 
-VERSION = 'V.0.2.a.20161125'
+VERSION = 'V.1.0.a.20161221'
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-- 
1.9.1

